# for soil

float32 soilhumidity
